benchmark/Benchmark/Compare.hs

```diff
@@ -0,0 +1,45 @@
+module Benchmark.Compare
```

Comment:
I think that this module can be safely renamed to `Main`. We already have `Main.hs` in this package, so need to create `Benchmark.Compare` as a separate module.